app/views/steps/case_details/edit.html.erb

```diff
@@ -0,0 +1,23 @@
+<% title t('.page_title') %>
+<% step_header(path: applications_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= govuk_error_summary(@form_object) %>
+    <h1 class="govuk-heading-xl">Case Details</h1>
+    <%= step_form @form_object do |f| %>
+      <%= f.govuk_text_field :ufn, width: 'one-third', autocomplete: 'off' %>
+      <%= f.govuk_text_field :main_offence, width: 'one-third', autocomplete: 'off' %>
```

Comment:
Worth having a look how I have done the autocomplete in the hearing details PR.

Can be done separately to PR if you want.

Reply:
later